nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -129,6 +129,7 @@ void Costmap2DROS::init()
   declare_parameter("rolling_window", rclcpp::ParameterValue(false));
   declare_parameter("track_unknown_space", rclcpp::ParameterValue(false));
   declare_parameter("transform_tolerance", rclcpp::ParameterValue(0.3));
+  declare_parameter("wait_for_transforms_timeout", rclcpp::ParameterValue(10.0));
```

Comment:
How about `initial_transform_timeout`?

Reply:
^ also needs a docs PR for navigation.ros.org adding it + a migration guide entry about the change. Please submit that for the next PR review cycle (I'll probably be able to merge the pair together)